@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -708,19 +715,15 @@ export class TiltViewClipPlanesEvaluator extends TopViewClipPlanesEvaluator {
         assert(projection.type !== ProjectionType.Spherical);
         const viewRanges = { ...this.minimumViewRange };
 
-        // Generally near/far planes are set to keep top/bottom planes intersection distance.
-        // Then elevations margins are applied. Here margins (min/max elevations) are meant to
-        // be defined as distance along the ground normal vector thus during camera
-        // tilt they may affect near/far planes positions differently.
+        // Find intersections of top/bottom frustum side's medians with the ground plane, taking
```

Comment:
what does `medians` mean in this context?

Reply:
removed